Source/WebCore/platform/graphics/filters/FilterOperations.h

```diff
@@ -37,20 +38,37 @@ struct BlendingContext;
 class FilterOperations {
     WTF_MAKE_FAST_ALLOCATED;
 public:
+    using const_iterator = Vector<Ref<FilterOperation>>::const_iterator;
+    using const_reverse_iterator = Vector<Ref<FilterOperation>>::const_reverse_iterator;
+    using value_type = Vector<Ref<FilterOperation>>::value_type;
+
     FilterOperations() = default;
-    WEBCORE_EXPORT explicit FilterOperations(Vector<RefPtr<FilterOperation>>&&);
+    WEBCORE_EXPORT explicit FilterOperations(Vector<Ref<FilterOperation>>&&);
+
+    FilterOperations(const FilterOperations&) = default;
+    FilterOperations(FilterOperations&&) = default;
+    FilterOperations& operator=(const FilterOperations&) = default;
+    FilterOperations& operator=(FilterOperations&&) = default;
```

Comment:
Why do we have to write this out explicitly?

Reply:
No. Not sure why I thought I did.